include/database.py

```diff
@@ -1,18 +1,47 @@
 # database.py
 #
 # This module provides a wrapper for basic database functionality.
+import sys
 import psycopg2
 
+
 def select(connectionString, sql, parameters):
+
     # Open the connection
-    connection = psycopg2.connect(connectionString)
-    cursor = connection.cursor()
+    try:
+        connection = psycopg2.connect(connectionString, connect_timeout = 5)
```

Comment:
Good, but double check to make sure the parameter will override any setting in the connection string. If so, add a comment so other developers know what's going on.

Reply:
Have you tested yet to make sure that the connection times out after 5 seconds and throws the correct exception? Comment still reflects what is in the YAML file.